### PR TITLE
[debian] Add non-PostgreSQL tableversion and dbpatch dependency alternatives

### PR DESCRIPTION
**Pull request by imincik**

### CODE REVIEW

**Comment by imincik** [comment]
@strk , ready to review once CI is finished

**Comment by strk** [comment]
CI fails with:
> dpkg-gencontrol: warning: can't parse dependency tableversion

Not sure what it is all about

**Comment by strk** [comment]
Why are `tableversion` and `dbpatch` in `Recommend` instead of a `Depend` ? Both `dbpatch-loader` and `table_version-loader`, included respectively in `dbpatch` and `tableversion` debian packages, are invoked by `linz-bde-schema-load`, which would be included in the `linz-bde-schema` debian package, thus I'd think `Depend` is more appropriate.

**Comment by imincik** [comment]
Great catch @strk , I made mistake. Will fix it now.

**Comment by strk** [comment]
CI fails because it's not installing `tableversion` and `dbpatch`  which would be now Required for installing linz-bde-schema. Should be an easy fix. Let me know if you want me to deal with it, in the test workflow (I'd install those packages ONLY as part as the packaging test)

**Comment by strk** [comment]
Ok I see you took another path but doing so you disabled the support for testing tableversion/dbpatch from their master branch. Shall this be decided separately ?
I've a less intrusive commit that could replace that change

**Comment by imincik** [comment]
@strk I was thinking if I should ask or not. What's the point of testing against tableversion and dbpatch master branches ? I thought it was just some workaround to some previous bug in tableversion or dbpatch.

**Comment by strk** [comment]
My patch, which you may want to try by removing your last commit:
```
commit 6d8c7227b14f718ea7acb9e8400f95a3b08bc135 (HEAD -> fix-tableversion-dbpatch-dep)
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Apr 8 10:56:47 2020 +0200

    Have packaging test deal with required dependencies

diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index 21dacb5..5e37f90 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -103,6 +103,9 @@ jobs:
     - name: Test packaging
       run: |
         make deb
+        # This deb package requires dbpatch and tableversion
+        # Dunno if we should ask dpkg to install anyway...
+        sudo apt-get install dbpatch tableversion
         sudo dpkg -i ../linz-bde-schema*.deb
     - name: Test publication
       run: |
```

**Comment by strk** [comment]
CI used to test multiple branches. The point is earlier catching of issues introduced with commits in their master branches. Ideally we'd be testing both packaged and in-development versions (release and master).

I guess there could be other reasons, like... a new version of tableversion/dbpatch requires changes in linz-bde-schema and we don't want to publish tableversion/dbpatch packages before having them tested with linz-bde-schema...

**Comment by imincik** [comment]
ok, reverted this change and pushed fix, but I see your advice is better. Will change it.

**Comment by imincik** [comment]
@strk , CI finally passed. Are you ok now to merge ?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -61,7 +61,9 @@ jobs:
             postgresql-${{ matrix.pg }}-pgtap \
             postgresql-${{ matrix.pg }}-postgis-${{ matrix.pgis }} \
             postgresql-${{ matrix.pg }}-postgis-${{ matrix.pgis }}-scripts \
-            postgresql-server-dev-${{ matrix.pg }}
+            postgresql-server-dev-${{ matrix.pg }} \
+            tableversion \
+            dbpatch
```
Comment (strk) [inline .github/workflows/test.yml:66]: Ok with the merge but note we did NOT need tableversion/dbpatch here (they are installed when needed, under `Test packaging` step). Should not harm, but it's preferrable, I think, to test the code in absence of those packages intially (so we also test that the "master" branches of tableversion/dbpatch repositories do install fine....

**Pull request merged**